diff/apply/apply.go

```diff
@@ -88,8 +89,12 @@ func (s *fsApplier) Apply(ctx context.Context, desc ocispec.Descriptor, mounts [
 	defer processor.Close()
 
 	digester := digest.Canonical.Digester()
+	reader := ioutil.NewConReader(processor)
```

Comment:
would you mind to provide cpu/memory usage about this? Thanks

Reply:
Added, has been updated. 
Please recheck.